tests/test_interfaces.py

```diff
@@ -113,7 +113,7 @@ def test_confignote_interface():
                     content={
```

Comment:
I think you need to add tests for all the changes you made in openreview_interface. py

Reply:
you need to add some tests for custom user demand computation. The new changes in openreview_interface need to be tested.

Reply:
I have added a new test and updated existing tests to test for the new changes.